src/check.h.in

```diff
@@ -673,7 +673,7 @@ CK_DLL_EXP void CK_EXPORT _ck_assert_failed(const char *file, int line,
 
 /* Number of digits after the decimal point to output via printf */
 #ifndef CK_FLOATING_DIG
-# define CK_FLOATING_DIG 6
+# define CK_FLOATING_DIG "6"
```

Comment:
I do not like an idea that precision might be a string. It gives users possibility to add hacks to Checks... And I am sure that somebody will use this possibility for their needs while it would lead problems afterwards. Writing 06, +6 would cause different situations too.
But using \#CK_FLOATING_DIG would cause more problems since it would be the same as string in fact.